### PR TITLE
fixed some errors in installation setup

### PR DESCRIPTION
**Pull request by FxKu**
I tried to install pg-trajectory but got a few errors. Most of them were easy to fix. Two things I found strange:
* There was a line of SQL without ; flying around were it shouldn't be:
```sql
SELECT (t1.tr).s_time, (t2.tr).s_time FROM t_life_25_point_small_1 t1, t_life_25_point_small_2 t2
```
* I couldn't find any function called t_type. From the comments it's easy to guess what it does, but before implementing it, I thought you could check your sources if you have simply forgotten to commit this function. So, in this PR I've simply commented it out.

### CODE REVIEW

File: model/TrajectoryDataModel.sql [hunk 1]
```diff
@@ -20,20 +20,20 @@ $BODY$
 DECLARE
   t trajectory;
 BEGIN
-    t.geom_type = t_type($1);
+    /*t.geom_type = t_type($1);
```
Comment (ahmetkucuk) [inline model/TrajectoryDataModel.sql]: @FxKu You just need to rename it to `tg_type` and It should work. Seems like we renamed `t_type` to `tg_type` but never commit the change here.

File: model/TrajectoryDataModel.sql [hunk 2]
```diff
@@ -20,20 +20,20 @@ $BODY$
...
     t.e_time = tg_end_time($1);
     t.s_time = tg_start_time($1);
     t.tr_data = array_sort($1);
-    IF array_length($1, 1) > 1 THEN
+    /*IF array_length($1, 1) > 1 THEN
         t.sampling_interval = (t.e_time - t.s_time) / (array_length($1, 1) - 1);
     ELSE
         t.sampling_interval = INTERVAL '-1 seconds';
-    END IF;
+    END IF;*/
```
Comment (ahmetkucuk) [inline model/TrajectoryDataModel.sql]: This is essential to run any distance function so we should not comment out this.

**Comment by FxKu** [comment]
Wow, better later then never :) `tg_type` it was, indeed. Thanks.
I found another bug, that produced an error in the SQL file, generated by `make install`.

**Comment by ahmetkucuk** [comment]
@FxKu  Thank you for the contribution! Sorry about being too late :)

In general, we should not make any changes in `pg_trajectory--0.0.1.sql`. We should create `pg_trajectory--0.0.2.sql`

**Pull request merged**